src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const part in actions) {
+    const action = actions[part];
+
+    if (action.type === 'addProperties') {
+      Object.assign(state, action['extraData']);
+    };
+
+    if (action.type === 'removeProperties') {
+      for (const removeElement in action.keysToRemove) {
+        for (const key in state) {
+          if (key === action.keysToRemove[removeElement]) {
+            delete state[key];
+          }
+        }
```

Comment:
as I see, here is should be just
```suggestion
      for (const removeElement in action.keysToRemove) {
        delete state[removeElement];
```